app/router.js

```diff
@@ -149,6 +150,7 @@ router.map(function() {
   this.route('orders', function() {
     this.route('new');
   });
+  this.route('pages', { path: '/*path' });
```

Comment:
Why did you create a new route ? Instead rename `not-found-catch` route. Only one wildcard route can exist at a given level.

Reply:
@niranjan94 But the ```not-found-catch``` has to render different template than the pages.

Reply:
Yes. And you need to handle that here. If the page doesn't match anything that is available, you need to render the not-found template.